cms/migrations/0017_create_and_setup_home_page.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.10 on 2020-06-09 10:12
+
+from django.db import migrations
+from django.contrib.contenttypes.models import ContentType
+from wagtail.core.models import Page, Site
+
+from cms.models import HomePage
+
+
+DEFAULT_HOMEPAGE_PROPS = dict(
+    title="MIT Bootcamps", tagline="Learn Innovation and Entrepreneurship from MIT"
+)
+DEFAULT_SITE_PROPS = dict(hostname="localhost", port=80)
+
+
+def create_home_page(apps, schema_editor):
+    """
+    Create the home page
+    """
+    site = Site.objects.filter(is_default_site=True).first()
+    valid_home_page = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(HomePage)
+    ).first()
+    root = Page.objects.get(depth=1)
+    if valid_home_page is None:
+        valid_home_page = HomePage(**DEFAULT_HOMEPAGE_PROPS)
+        root.add_child(instance=valid_home_page)
+        valid_home_page.refresh_from_db()
+    if site is None:
+        Site.objects.create(
+            is_default_site=True, root_page=valid_home_page, **DEFAULT_SITE_PROPS
+        )
+    elif site.root_page is None or site.root_page != valid_home_page:
+        site.root_page = valid_home_page
+        site.save()
+    wagtail_default_home_page = Page.objects.filter(
+        depth=2, content_type=ContentType.objects.get_for_model(Page)
+    ).first()
+    if wagtail_default_home_page is not None:
+        wagtail_default_home_page.delete()
+
+
+def remove_home_page(apps, schema_editor):
+    """
+    Remove the home page
+    """
+    home_page = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(HomePage)
+    ).first()
+    home_page_descendants = home_page.get_descendants()
+    for descendant in home_page_descendants:
+        descendant.delete()
+    home_page.delete()
```

Comment:
This is incorrect. You need to create a new default wagtail home page with title "Welcome to your new Wagtail site!" and also transfer any existing home page descendants under the default home page before you delete the one created earlier by applying this migration. Point being, if I apply and then rollback this migration my CMS and database should remain the same.